### DISCUSSION TITLE
Recommended AWS instance type for running (long) Cypress runs

### DISCUSSION THREAD

**Discussion by penzeliz** [cypress-io/cypress#9356] [Docker]
Hi All,

I'd like to get a view of what type of AWS instances people are using or is recommended when running Cypress tests in a Docker container.

Currently, we're on a `c5.2xlarge` and we have the following issues and the tests are still running relatively slow - compared to our MacBooks, which _should be_ less performant than a 2xlarge, am I right?

(Sometimes tests are hanging too - could be a different issue, I know there are a few bug reports around this... just adding it to the bucket :) We also have `--disable-dev-shm-usage` in Chrome options, and using `cypress/browsers:node12.14.1-chrome85-ff81` - but it's been like this since forever)

Regards,

Zoltan

**Comment by AlexSdet** [comment]
Curious about the same thing!

**Comment by minh-gfg** [comment]
Same issue, cypress was smooth on local but some time hung, extremely slow on `c6a.2xlarge` runner
